.travis.yml

```diff
@@ -18,6 +18,8 @@ jobs:
         - pip3 install pyyaml
         - pip3 install aura.tar.gz
       script:
+        - chmod +x ./scripts/check-asciidoctor-build.sh
+        - ./scripts/check-asciidoctor-build.sh
```

Comment:
**issue:** From the example even if this script fail, build doesn't stop. PS https://docs.travis-ci.com/user/job-lifecycle/#breaking-the-build

So maybe separate it out:
```yml
before_script:
  - chmod +x ./scripts/check-asciidoctor-build.sh
  - ./scripts/check-asciidoctor-build.sh
script:
  - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.13 --no-upstream-fetch && python3 makeBuild.py
```

Reply:
I'm not sure if it will stop all other build stages for different distros? Maybe check, otherwise we can must update travis yml to first pass this check and then only proceed for other stages.

Reply:
Edit: You are right. Updating...